Checks/PWR043/README.md

```diff
@@ -84,24 +89,93 @@ for (int i = 0; i < n; i++) {
 ```
 
 The first loop (line 1) is not performance critical, since it is not nested. On
-the other hand, the second loop (line 4) is performance critical, since it
+the other hand, the second loop (line 5) is performance critical, since it
 contains the loop nest.
 
-Fortunately, the loop nest is now perfectly nested, so that loop interchange is
-applicable. The final result looks like this (note the order of the nested loops
-is now `ji` instead of the original order `ij`):
+Fortunately, the loop nest is now perfectly nested, making loop interchange
+applicable. The final result has the nested loops in `ji` order instead of the
+original `ij` order:
 
 ```c
 for (int i = 0; i < n; i++) {
   b[i] = 0.0;
 }
+
 for (int j = 0; j < n; j++) {
   for (int i = 0; i < n; i++) {
     b[i] += a[j][i];
   }
 }
 ```
 
+#### Fortran
+
+Have a look at the following code:
+
```

Comment:
nit: unnecessary, I would just remove this sentence

Reply:
Indeed, it doesn't add any value. For the sake of brevity, [I've removed](https://github.com/codee-com/open-catalog/compare/70d218ee94a154404de7f2047bac58ee0689635a..53da9a7b92e35919f1543c12f7650952d5c94324) the sentence from all entries where it is repeated.